spec/api.md

```diff
@@ -0,0 +1,4019 @@
+# API
+
+Related [Confluence page](https://govstack-global.atlassian.net/wiki/spaces/DEMO/pages/179601480/Registration#OpenIMIS-resources)
+
+## User login
+
+User/Civil servant logs in. To create new record the user must have certain role
+
+POST `/api_fhir_r4/login/`
+
+```json
+{
+  "username": "RHOS0011",
+  "password": "RHOS0011"
+}
+```
+
+Response
+
+```json
+{
+  "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6IkFkbWluIiwiZXhwIjoxNjc5MDU0MTQxLCJvcmlnSWF0IjoxNjc4OTY3NzQxfQ.P-ZKhTWf7IbuWkWnKknSVhGDprcRq4V10-ged33IU2E",
+  "exp": 1679054141
+}
+```
+
+
+## Provide Citizen personal data
+Civil servant opens the registration form to fill the beneficiary registration form. 
+Civil Servant enters citizens personal ID. Citizens personal data is pulled from the CR and form filled automatically
+
+POST `/api_fhir_r4/Patient/`
+
+
+```json
+{
+  "resourceType": "OperationOutcome",
+  "issue": [
+    {
+      "severity": "fatal",
+      "code": "exception",
+      "details": {
+        "text": "You do not have permission to perform this action."
```

Comment:
I checked necessary API endpoints.
Looks good overall, but nice to know how to get permission?

Reply:
To get permission need to use specific backup.
https://github.com/GovStackWorkingGroup/sandbox-open-imis/pull/2/commits/d451741b966e2bece44e2f83cb26c5d5ed2d6f4b